R/loocv.R

```diff
@@ -191,8 +191,14 @@ is.baggr_cv <- function(x) {
 #' Given multiple [loocv] outputs, calculate differences in their expected log
 #' predictive density.
 #'
-#' @param x An object of class `baggr_cv` or a list of such objects.
+#' @param x A list of `baggr_cv` objects, with a minimum of 2 objects required for comparison.
```

Comment:
They can be named, right?

Reply:
The `baggr_cv` objects can be named, though I'm unsure if those names are passed as attributes to `loo_compare`. I was curious if we wanted to change this so that rather than using Model 1 through Model N, this uses the names of the models being compared so that the output of `loo_compare(cv1,cv2,cv3)` would be 
```
                   ELPD  ELPD SE
cv1 - cv2          A              C
cv1 - cv3          B              D
```

Reply:
No, you shouldn't deparse names like that, you'd want to do `loo_compare("Model 1" = cv1, "Other" = cv2)`. See `baggr_compare` for inspiration. Do you reckon you could implement that by way of analogy?

Reply:
Sure thing! I'll take care of that.